composer.json

```diff
@@ -32,9 +32,6 @@
 		}
 	},
 	"autoload": {
-		"files" : [
-			"SemanticFormsSelect.php"
-		],
```

Comment:
So the extension will no longer work when you install it via Composer on an older MW version?!